sound/soc/codecs/rt700.c

```diff
@@ -1134,6 +1134,9 @@ int rt700_io_init(struct device *dev, struct sdw_slave *slave)
 		return 0;
 
 	if (rt700->first_init) {
+		/* make cache_dirty */
+		regmap_write(rt700->regmap, 0x4f12, 0x91);
```

Comment:
There is an API for this, no?

void regcache_mark_dirty(struct regmap *map)

Reply:
I think about this again. Sorry, please ignore this PR. 
One month ago, I didn't consider the situation that the codec power may disable in suspend mode. Therefore, I removed the regcache_mark_dirty API from the suspend function.
This API will set two flags including
map->cache_dirty = true;
map->no_sync_defaults = true;
At that time, I didn't want the no_sync_defaults flag to set to true.
In the resume function, it just syncs the register range we want. 

I will make the new patch for this and make it reasonable.

Reply:
ok, let us know when the new version is ready.
I think once the interfaces changes are in Mark's tree we need to upstream all the codec drivers and get more feedback.